src/pages/components/content-section.mdx

```diff
@@ -0,0 +1,132 @@
+unwanted page banding experience (page banding means that constrasting full-width backgrounds are alternating in a web
+page, interrupting the reading experience for the user).
+
+#### When not to use
+
+Do not use a content section to only separate content items, or other related basic content items (like paragraphs)
+inside of a web page. Use a content group instead.
+
+## Behaviors
+
+The content section elements are persistent throughout the online experience. It is fully responsive and changes how
+elements are displayed based on the browser size.
+
+#### Desktop breakpoints (max, xlg, and lg)
+
+If the browser is wider than the max breakpoint of 1584px, the content section and all other page content will center
+and extra margin will appear on either side of the page layout. The content section heading, description and link CTA
+will take the first four (out of 16) columns of the grid, while the children container will take the other 12 columns.
+
+<Row>
+
+<Column colMd={4} colLg={12}>
+
+![Content section at xlg 1312px breakpoint encapsulating a content block simple](../../images/component/content-section/component--content-section--behavior-desktop.png)
+
+<Caption>Content section at xlg 1312px breakpoint encapsulating a content block simple</Caption>
+
+</Column>
+
+</Row>
+
+#### Md and sm breakpoints
+
+At medium and small breakpoints, the content section will add a significant change: the children container will move
+above the link CTA text link. See the image below.
+
+![A visual example of the content group component at mobile breakpoints](../../images/component/content-section/component--content-section--behavior-mobile.png)
+
+<Caption>Content section at md 672px and sm 320px breakpoints encapsulating a content block simple</Caption>
+
+### Showcase
+
+This is a showcase of a content section that encapsulates a content block simple, or, in turn, a card block with four
+cards.
+
+![A visual example of the content block component](../../images/component/content-section/component--content-section--showcase2.gif)
+
+<Caption>Showcase of a content section component</Caption>
+
+## Tips and techniques
+
+The content block elements are persistent throughout the online experience. It is fully responsive and changes how
+elements are displayed based on the browser size.
+
+- Important: keep in mind that on medium and small breakpoints, the children container is positioned between the content
+  section description and the CTA text link.
+- Each type of content unit (content section, content block, content group or content item, and their many options)
+  usually will have a certain type of CTA as the last element in the component architecture.
+- Try to avoid CTA stacking at the end of a content section. If multiple CTAs are present, use different types. Example:
+  a card link from a content block and a CTA text link as the final element of the content section.
```

Comment:
```suggestion
It's important to keep in mind that on medium and small breakpoints, the child container is positioned between the Content section description and the CTA text link.

Each type of content unit (content section, content block, content group or content item, and their many options) will usually have a certain type of CTA as the last element in the component architecture.

Try to avoid CTA stacking at the end of a Content section. If multiple CTAs are present, use different types. For example, a card link from a Content block and a CTA text link as the final element of the Content section.
```